### PR TITLE
Fi build and readme

### PR DESCRIPTION
**Pull request by vsmalladi**
* Fix some of the description in readme and tags.

* Fix build to run based on requirements of plugins

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
 
+The [Cloud Work Stream](https://ga4gh.cloud) helps the genomics and health communities take full advantage of modern
+cloud environments.
+Our initial focus is on “bringing the algorithms to the data”, by creating standards for defining, sharing, and
+executing portable workflows.
+
+We work with platform development partners and industry leaders to develop standards that will facilitate
+interoperability.
+
+Cloud Work Stream
```
Comment (uniqueg) [inline README.md:24]: If I'm not mistaken, it looks like we're gonna have two sections with the same name. If so, please merge into a single section.

**Review by uniqueg — CHANGES_REQUESTED** [review]
Looks good mostly, but there's one section that seems to need some cleanup.

File: README.md (hunk 1 above)
Comment (vsmalladi) [reply README.md]: @uniqueg fixed

**Review by uniqueg — APPROVED** [review]
LGTM

**Pull request merged**